src/shared/dropdown-list/dropdown-list.styles.tsx

```diff
@@ -245,11 +277,23 @@ export const ResultStateContainer = styled.div`
     align-items: center;
 `;
 
-export const ResultStateText = styled(Text.Body)``;
+export const ResultStateText = styled.text<ResultStateProps>`
```

Comment:
I don't think text is a standard html element? render a div instead

```suggestion
export const ResultStateText = styled.div<ResultStateProps>`
```